<launch>
  <!-- send hand urdf to param server -->
  <include file="$(find sr_description)/loaders/load_hand_model_robot_description.launch" />

  <!-- the semantic description that corresponds to the URDF-->
  <param name="robot_description_semantic" textfile="$(find hand_kinematics)/launch/shadowhand_motor.srdf" /> 

  <test test-name="hand_kinematics_plugin_test" pkg="hand_kinematics" type="test_kinematics_as_plugin" name="right_hand_kinematics" time-limit="180">
   <param name="tip_name" value="fftip" />     
   <param name="root_name" value="palm" />  
   <param name="all_fingers/maxIterations" type="int" value="1000"/>
   <param name="all_fingers/epsilon" type="double" value="0.001"/>
   <param name="search_discretization" value="0.025" />  
   <param name="number_fk_tests" value="100" />  
   <param name="number_ik_tests" value="100" />  
   <param name="number_fk_tests_with_callbacks" value="100" />  

   <param name="plugin_name" value="hand_kinematics/HandKinematicsPlugin" />  
  </test>

</launch>
